### DISCUSSION TITLE
Help to mitigate CVE-2021-20323 Keycloak XSS issue

### DISCUSSION THREAD

**Discussion by bjornerikolsen** [keycloak/keycloak#15527] [Q&A] [answered]
I’m running Keycloak 15 that have a XSS security vulnerability [2013577 – (CVE-2021-20323) CVE-2021-20323 keycloak-services: POST based reflected Cross Site Scripting vulnerability 2](https://bugzilla.redhat.com/show_bug.cgi?id=2013577) Reflected XSS on clients-registrations endpoint. It’s fixed in 18.0.0, but I cannot upgrade to 18.0.0 (that would be the best I know). So I’m looking at workarounds to mitigate. But how? This? https://www.keycloak.org/2021/12/cve

**Comment by abstractj** [comment, marked as answer]
@bjornerikolsen please upgrade to the latest release, our team already resolved this issue. For the complete list of advisories, please check https://github.com/keycloak/keycloak/security/advisories?state=published
